### DISCUSSION TITLE
Disable docs generation on a particular entry point?

### DISCUSSION THREAD

**Discussion by hemna** [fastapi/fastapi#11209] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
I would like to disable the automatic docs generation on a endpoint that isn't really part of my external API.  I'm using FastAPI to serve up my API as well as the website for my app.

For example, this serves up my index.html page, and there is no reason it should be in the 
/docs url.  How can I disable this URL/endpoint from being generated in the /docs url?

@app.get("/", response_class=HTMLResponse)
async def get(request: Request):
    services = APRSServices()
    return templates.TemplateResponse(
        request=request,
        name="index.html",
        context={"request": request,
                 "services": services}
    )
```

### Description

How can I disable docs generation on a particular endpoint/url ?

### Operating System

Linux

### Operating System Details

doesn't matter

### FastAPI Version

└─> python -c "import fastapi; print(fastapi.__version__)" 0.110.0

### Pydantic Version

└─> python -c "import pydantic; print(pydantic.version.VERSION)" 1.10.14

### Python Version

└─> python --version Python 3.10.13

**Comment by JavierSanchezCastro** [comment, marked as answer]
Just add 

`include_in_schema=False`

like
```python3
@app.get("/", response_class=HTMLResponse, include_in_schema=False)
async def get(request: Request):
    services = APRSServices()
    return templates.TemplateResponse(
        request=request,
        name="index.html",
        context={"request": request,
                 "services": services}
    )
```
